react-interactive/src/GenericSlice.tsx

```diff
@@ -474,12 +475,13 @@ export default class GenericSlice<T extends U> {
                     if (this.actionErrorDependency !== null)
                         this.actionErrorDependency(state as IPagedState<T>,`${name}/Page${name}`, action.meta.arg, action.meta.requestId)
                 });
-                builder.addCase(dBPage.fulfilled, (state: WritableDraft<IPagedState<T>>, action: PayloadAction<any, string,  {arg: { page?: number, filter:  Search.IFilter<T>[], sortfield?: keyof T, ascending?: boolean}, requestId: string},never>) => {
+                builder.addCase(dBPage.fulfilled, (state: WritableDraft<IPagedState<T>>, action: PayloadAction<any, string, { arg: { page?: number, filter: Search.IFilter<T>[], sortfield?: keyof T, ascending?: boolean, parentID?: number | string }, requestId: string},never>) => {
                     state.ActivePagedID = state.ActivePagedID.filter(id => id !== action.meta.requestId);
                     state.PagedStatus = 'idle';
                     state.TotalPages = action.payload.NumberOfPages;
-                    state.SearchResults = JSON.parse(action.payload.Data);
+                    state.PagedData = JSON.parse(action.payload.Data);
                     state.Filter = action.meta.arg.filter;
+                    state.ParentID = action.meta.arg.parentID !== undefined ? action.meta.arg.parentID : null; 
```

Comment:
See comments above. You are changing the `ParentID` in `state` but not updating the `fetch` Data.
Why do we actually need to make this change? If the `ParentID` is set via an initial `Fetch`  call which should already have happened anyway, adding a `dBPage` call should have respected the `ParentID`. If you really want to avoid a `Fetch` call you will need to include the `ParentID` in the `dBPage` but set and respect `ParentID` as appropriate and reset the `data` state back to `uninitated` to avoid causing some pages to stop loading the data when being pulled up.